src/Halogen/Aff/Driver/Eval.purs

```diff
@@ -48,9 +47,7 @@ handleLifecycle lchs f = do
   result <- liftEff f
   { initializers, finalizers } <- liftEff $ readRef lchs
   forkAll finalizers
-  -- No need to par/fork initializers here as there's only ever zero or one at
-  -- this point, due to the squashing at each level of the component hierarchy.
-  sequence_ initializers
+  parSequence_ initializers
```

Comment:
The comment was only true due to the sequence-squashing that occurred before. Here is where our 900 initializers run in the example. So yeah, we want them to run in parallel here.